app/views/groups/group_links/_invited_groups.html.erb

```diff
@@ -1,29 +1,53 @@
-<table class="min-w-full table-fixed dark:divide-gray-600">
-  <thead class="text-gray-700 bg-gray-50 dark:bg-gray-700 dark:text-gray-400">
-    <tr>
-      <th scope="col" class="px-6 py-3 text-left"><%= t(:"groups.group_links.index.table_header.group") %></th>
-      <th scope="col" class="px-6 py-3 text-left"><%= t(:"groups.group_links.index.table_header.source") %></th>
-      <th scope="col" class="px-6 py-3 text-left"><%= t(:"groups.group_links.index.table_header.access_granted") %></th>
-      <th scope="col" class="px-6 py-3 text-left"><%= t(:"groups.group_links.index.table_header.access_level") %></th>
-      <th scope="col" class="px-6 py-3 text-left"><%= t(:"groups.group_links.index.table_header.expiration") %></th>
-      <th scope="col" class="px-6 py-3 text-right"><%= t(:"groups.group_links.index.table_header.action") %></th>
-    </tr>
-  </thead>
-  <tbody
-    id="invited-groups-table-body"
+<div class="table-container relative overflow-x-auto" data-turbo-temporary>
+  <table
     class="
-      bg-white
-      divide-y
-      divide-gray-200
-      dark:bg-gray-800
-      dark:divide-gray-700
+      w-full
+      text-sm
+      text-left
+      rtl:text-right
+      text-slate-500
+      dark:text-slate-400
+      dark:divide-gray-600
+      whitespace-nowrap
     "
   >
-    <% @namespace_group_links.each do |namespace_group_link| %>
-      <%= render partial: "namespace_group_link",
-      locals: {
-        namespace_group_link: namespace_group_link
-      } %>
-    <% end %>
-  </tbody>
-</table>
+    <thead
+      class="
+        text-xs
+        text-slate-700
+        uppercase
+        bg-slate-50
+        dark:bg-slate-700
+        dark:text-slate-400
+      "
+    >
+      <tr>
+        <th scope="col" class="px-6 py-3 text-left"><%= t(:"groups.group_links.index.table_header.group") %></th>
+        <th scope="col" class="px-6 py-3 text-left"><%= t(:"groups.group_links.index.table_header.source") %></th>
+        <th scope="col" class="px-6 py-3 text-left"><%= t(:"groups.group_links.index.table_header.access_granted") %></th>
+        <th scope="col" class="px-6 py-3 text-left"><%= t(:"groups.group_links.index.table_header.access_level") %></th>
```

Comment:
For the access level and expiration columns I think we should set the column width. If the window is resized the columns collapse (see below) This comment applies to the projects invited_groups, members, and group members pages. 

![image](https://github.com/phac-nml/irida-next/assets/25466211/4e914db7-124f-44ca-b6bf-c1f6ba763294)

Reply:
Fixed in [affbd72](https://github.com/phac-nml/irida-next/pull/588/commits/affbd72ca030866300087457173c5410c04bd14c)